docs/docs/events/index.md

```diff
@@ -0,0 +1,165 @@
+# Getting Started with Detecting
```

Comment:
need to move events from contributing here

Reply:
done